packages/components/button/src/button.stories.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import CoreButton from './button';
+import Box from '../../box/src/box';
 
 export default {
```

Comment:
let's try to use [controls plugin](https://storybook.js.org/docs/react/essentials/controls) to give users a chance to try different combinations of props